Base/res/html/error.html

```diff
@@ -13,7 +13,7 @@
     </head>
     <body>
         <header>
-            <img src="file:///res/icons/32x32/msgbox-warning.png" alt="Warning" width="24" height="24">
+            <img src="@icon_path@" alt="Warning" width="24" height="24">
```

Comment:
Seems overly complicated to add an entire new setter/member/CLI option for this. `../icons32x32/msgbox-warning.png` should do the trick, no?

Reply:
I've just tested your suggestion, what I can see is that while the page is from `res/html/error` the loader will attempt to load it relative to the errored page. e.g. `github.com/404` will attempt load `github.com/404/../icons32x32/msgbox-warning.png`
![image](https://user-images.githubusercontent.com/36214028/213178383-e7e8e9be-31ba-4d20-9b99-640327594ef0.png)

Reply:
Ah, that's because we do `load_html(generator.as_string_view(), failed_url);`, which seems extremely sketchy anyway (probably to trick the browser into thinking that the failed URL is still the document's URL). Needs more discussion in #browser, I appreciate the fix but am not happy to paper over it like that.

Reply:
Perfect, should I just remove that commit? I don't think ill have the knowledge to tackle it any other way atm.